pep-0004.txt

```diff
@@ -24,284 +24,17 @@ At the same time, removing a module from the distribution is
 difficult, as it is not known in general whether anybody is still
 using it.  This PEP defines a procedure for removing modules from the
 standard Python library.  Usage of a module may be 'deprecated', which
-means that it may be removed from a future Python release.  The
-rationale for deprecating a module is also collected in this PEP.  If
-the rationale turns out faulty, the module may become 'undeprecated'.
+means that it may be removed from a future Python release.
 
 
 Procedure for declaring a module deprecated
 ===========================================
 
-Since the status of module deprecation is recorded in this PEP,
-proposals for deprecating modules MUST be made by providing a change
-to the text of this PEP.
+To remove a top-level module/package from the standard library, a PEP
+is required. The deprecation process outlined in :pep:`387`.
 
-A proposal for deprecation of the module MUST include the date of the
-proposed deprecation and a rationale for deprecating it.  In addition,
-the proposal MUST include a change to the documentation of the module;
-deprecation is indicated by saying that the module is "obsolete" or
-"deprecated".  The proposal SHOULD include a patch for the module's
-source code to indicate deprecation there as well, by raising a
-DeprecationWarning.  The proposal MUST include patches to remove any
-use of the deprecated module from the standard library.
-
-It is expected that deprecated modules are included in the Python
-release that immediately follows the deprecation; later releases may
-ship without the deprecated modules.
-
-
-For modules existing in both Python 2.7 and Python 3.5
-------------------------------------------------------
-
-In order to facilitate writing code that works in both Python 2 & 3
-simultaneously, any module that exists in both Python 3.5 and
-Python 2.7 will not be removed from the standard library until
-Python 2.7 is no longer supported as specified by :pep:`373`. Exempted
-from this rule is any module in the idlelib package as well as any
-exceptions granted by the Python development team.
-
-
-Procedure for declaring a module undeprecated
-=============================================
-
-When a module becomes deprecated, a rationale is given for its
-deprecation.  In some cases, an alternative interface for the same
-functionality is provided, so the old interface is deprecated.  In
-other cases, the need for having the functionality of the module may
-not exist anymore.
-
-If the rationale is faulty, again a change to this PEP's text MUST be
-submitted.  This change MUST include the date of undeprecation and a
-rationale for undeprecation.  Modules that are undeprecated under this
-procedure MUST be listed in this PEP for at least one major release of
-Python.
-
-
-Obsolete modules
```

Comment:
Again, I think it's better to keep this PEP around as a historical document and just declare it to be superseded by 387.

Reply:
Very much agreed as well